README.md

```diff
@@ -31,16 +31,50 @@ sudo npm i -g pnpm
    - While decky-loader primarily targets Steam Deck hardware so keep this in mind when developing your plugin.
 4. If you want an all encompassing demonstration of decky-frontend-lib's capabilites check out [decky-playground](https://github.com/SteamDeckHomebrew/decky-playground). It shows off almost all of decky-frontend-lib's features.
 
-#### Other important information
+#### Development and testing
 
-Everytime you change the frontend code (`index.tsx` etc) you will need to rebuild using the commands from step 2 above or the build task if you're using vscode or a derivative.
+### Re-building
 
-Note: If you are receiving build errors due to an out of date library, you should run this command inside of your repository:
+Every time you change the frontend code (`index.tsx` etc) you will need to rebuild using the commands from step 2 above.
+
+### Automation with VS Code
+
+If you're using vscode or a derivative you can use the tasks included in the `.vscode` template project to automate many parts of the process of building and deploying your plugin changes.
+
+In order to run these pre-made tasks: 
+
+- open VSCode
+- [install the "Task Runner" extension](https://marketplace.visualstudio.com/items?itemName=forbeslindesay.forbeslindesay-taskrunner) from the VS Code extension panel
```

Comment:
Users can run tasks without this extension, we'd rather not recommend dependencies until we have a shellscript equivalent to our vscode tasks.